pkgs/applications/networking/p2p/tremotesf/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, cmake
+, pkg-config
+, mkDerivation
+, fetchurl
+, qtbase
+, qttools
+, kwidgetsaddons
+, kwindowsystem
+, fmt
+, libpsl
+, cxxopts
+, wrapQtAppsHook
+}:
+
+mkDerivation rec {
+  pname = "tremotesf";
+  version = "2.3.0";
+
+  doCheck = true;
```

Comment:
This is not needed because it's the implicit default

Reply:
I have removed it but interestingly now I don't see tests running when I build locally. I don't think I have changed anything but I suppose the CI/build system defaults are different than mine?

Reply:
> Just a few nitpicks but overall quite a good PR heart

Thank you for the review and the suggestions. I know this can be a thankless task